engine/src/main/scala/cromwell/engine/workflow/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -0,0 +1,165 @@
+package cromwell.engine.workflow
+
+import java.nio.file.Paths
+
+import akka.actor.{Actor, Props}
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.scalalogging.LazyLogging
+import cromwell.engine._
+import cromwell.engine.backend.runtimeattributes.CromwellRuntimeAttributes
+import cromwell.engine.backend.{Backend, BackendType, CromwellBackend}
+import cromwell.engine.workflow.MaterializeWorkflowDescriptorActor.{MaterializationFailure, MaterializationSuccess}
+import cromwell.util.TryUtil
+import spray.json._
+import wdl4s._
+
+import scala.language.postfixOps
+import scala.util.{Failure, Success, Try}
+import scalaz.Scalaz._
+import scalaz.Validation.FlatMap._
+
+object MaterializeWorkflowDescriptorActor {
```

Comment:
@scottfrazer raises a good point. Ideally actors should be thought of more as functions manipulating data as part of a chain of events and less as objects w/ state (but obviously they can do either). This is more the former.